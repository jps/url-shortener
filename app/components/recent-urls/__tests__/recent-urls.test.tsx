import { render } from "@testing-library/react";
import { RecentUrls } from "../";

describe("RecentUrls should", () => {
  it("render as expected", () => {
    const { asFragment } = render(
        <RecentUrls urls={["https://jspenc.com/238748", "https://jspenc.com/rzo87mwn"]}  />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
