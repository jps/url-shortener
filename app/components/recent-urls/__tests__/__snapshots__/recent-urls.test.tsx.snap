// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecentUrls should render as expected 1`] = `
<DocumentFragment>
  <div
    class="recent-urls"
  >
    <h2>
      Recent URLs
    </h2>
    <ul
      class="recent-urls__list"
    >
      <li
        class="recent-urls__list-item"
      >
        <a
          href="https://jspenc.com/238748"
        >
          https://jspenc.com/238748
        </a>
      </li>
      <li
        class="recent-urls__list-item"
      >
        <a
          href="https://jspenc.com/rzo87mwn"
        >
          https://jspenc.com/rzo87mwn
        </a>
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
